src/main/scala/com.snowplowanalytics.sauna/observers/AmazonS3Observer.scala

```diff
@@ -53,7 +54,7 @@ class AmazonS3Observer(s3: S3, sqs: SQS, queue: Queue) extends Actor with Observ
 
```

Comment:
^^ this actor needs to be a singleton and to run in its own thread pool with size 1